qcelemental/tests/test_importing.py

```diff
@@ -73,20 +73,6 @@ def test_which_f_raisemsg():
     assert str(e.value).endswith("Command 'evills' not found in envvar PATH. Install `evills`.")
 
 
-def test_which_f_raise():
-    with pytest.raises(ModuleNotFoundError) as e:
-        qcel.util.which('evills', raise_error=True)
-
-    assert str(e.value).endswith("Command 'evills' not found in envvar PATH.")
-
-
-def test_which_f_raisemsg():
-    with pytest.raises(ModuleNotFoundError) as e:
-        qcel.util.which('evills', raise_error=True, raise_msg='Install `evills`.')
-
-    assert str(e.value).endswith("Command 'evills' not found in envvar PATH. Install `evills`.")
```

Comment:
what's wrong with these tests?

Reply:
Duplicates of the ones that were above them.

https://github.com/MolSSI/QCElemental/blob/9f3bbf0f4614e1b66ddd7e8df74ccb7be5a72ac3/qcelemental/tests/test_importing.py#L62-L71